specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2022-12-01-preview/ContentPackages.json

```diff
@@ -0,0 +1,399 @@
+          "description": "Flag indicates if this package is in preview."
+        },
+        "isFeatured": {
+          "$ref": "./common/ContentCommonTypes.json#/definitions/metadataTrueFalseFlag",
+          "description": "Flag indicates if this package is among the featured list."
+        },
+        "version": {
+          "$ref": "./common/ContentCommonTypes.json#/definitions/metadataVersion",
+          "description": "the latest version number of the package"
+        },
+        "displayName": {
+          "$ref": "./common/ContentCommonTypes.json#/definitions/metadataDisplayName",
+          "description": "The display name of the package"
+        },
+        "description": {
+          "description": "The description of the package",
+          "type": "string"
+        },
+        "source": {
+          "$ref": "./common/ContentCommonTypes.json#/definitions/metadataSource",
+          "description": "The source of the package"
+        },
+        "author": {
+          "$ref": "./common/ContentCommonTypes.json#/definitions/metadataAuthor",
+          "description": "The author of the package"
+        },
+        "support": {
+          "$ref": "./common/ContentCommonTypes.json#/definitions/metadataSupport",
+          "description": "The support tier of the package"
+        },
+        "dependencies": {
+          "$ref": "./common/ContentCommonTypes.json#/definitions/metadataDependencies",
+          "description": "The support tier of the package"
+        },
+        "providers": {
+          "$ref": "./common/ContentCommonTypes.json#/definitions/metadataProviders",
+          "description": "Providers for the package item"
+        },
+        "firstPublishDate": {
+          "$ref": "./common/ContentCommonTypes.json#/definitions/metadataFirstPublishDate",
+          "description": "first publish date package item"
+        },
+        "lastPublishDate": {
+          "$ref": "./common/ContentCommonTypes.json#/definitions/metadataLastPublishDate",
+          "description": "last publish date for the package item"
+        },
+        "categories": {
+          "$ref": "./common/ContentCommonTypes.json#/definitions/metadataCategories",
+          "description": "The categories of the package"
+        },
+        "threatAnalysisTactics": {
+          "$ref": "./common/ContentCommonTypes.json#/definitions/metadataThreatAnalysisTactics",
+          "description": "the tactics the resource covers"
+        },
+        "threatAnalyticsTechniques": {
+          "$ref": "./common/ContentCommonTypes.json#/definitions/metadataThreatAnalysisTechniques",
+          "description": "the techniques the resource covers, these have to be aligned with the tactics being used"
+        },
+        "icon": {
+          "$ref": "./common/ContentCommonTypes.json#/definitions/metadataIcon",
+          "description": "the icon identifier. this id can later be fetched from the content metadata"
+        },
+        "packageItems": {
```

Comment:
>packageItems [](http://example.com/codeflow?start=9&length=12)

why is this type not defined explicitly?

Reply:
blocking for this comment to be resolved

Reply:
Return the JSON string instead.